src/Form/Builder/ChoiceFacetFormBuilder.php

```diff
@@ -16,6 +17,13 @@ public function build(FormBuilderInterface $builder, Facet $facet, array $values
         $keys = array_keys($values);
         $choices = array_combine($keys, $keys);
 
+        /** @var class-string<ChoiceValuesSorterInterface> $sorter */
+        $sorter = $facet->sorter;
+        if ($facet->sorter !== null) {
+            /** @var array $choices */
+            $choices = (new $sorter())->sort($choices);
```

Comment:
Could you add a todo that in the future sorters should be services marked with `setono_sylius_meilisearch_facet_value_sorter`? Then in future they could be fetched from a `FacetValueSorterRegistry` or something and people would be able to do all kinds of magic when sorting.

I think for now this solution is fine 👍

Reply:
Maybe let's add an issue about it to the repo, so it's not lost?

Reply:
I will do that :)